Library/Formula/hfstospell.rb

```diff
@@ -0,0 +1,23 @@
+class Hfstospell < Formula
+  desc "Helsinki Finite-State Technology ospell"
+  homepage "http://www.ling.helsinki.fi/kieliteknologia/tutkimus/hfst/"
+  url "https://downloads.sourceforge.net/project/hfst/hfst/archive/hfstospell-0.3.0.tar.gz"
+  sha256 "07b5b368882cac2399edb1bb6e2dd91450b56f732c25413a19fcfe194342d70c"
+
+  depends_on "pkg-config" => :build
+  depends_on "libarchive"
+
+  def install
+    # Libxml++ headers do not seem to compile on El Capitan
```

Comment:
I don't know. Google didn't find anything relevant and I don't know if this is more an upstream or Homebrew issue, but on my machine the simplest program that fails to compile is this: 

```#include <libxml++/libxml++.h>
int main(void) { return 0; }
``` compiled with `g++ -Wall  $(pkg-config --cflags libxml++-2.6) foo.cc`. With `-std=c++11` the code compiles.